src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/begrunnelser/IBegrunnelse.kt

```diff
@@ -15,6 +15,10 @@ sealed interface IBegrunnelse {
 
     fun enumnavnTilString(): String
 
+    fun støtterFritekst(sanityBegrunnelser: List<SanityBegrunnelse>) =
+        sanityBegrunnelser.firstOrNull { it.apiNavn == this.sanityApiNavn }?.støtterFritekst ?: false ||
```

Comment:
`sanityBegrunnelser.firstOrNull { it.apiNavn == this.sanityApiNavn }?.støtterFritekst == true`